### DISCUSSION TITLE
Script commands for position of GUI elements?

### DISCUSSION THREAD

**Discussion by rayvd1** [Stellarium/stellarium#2565] [Q&A] [answered]
Hello,
Are there any script commands that would allow the script to control the placement of various GUI elements (like the date/time window, for example)? Or at least to report the pixel location and hide/show the window?

Thanks!
Ray

**Comment by gzotti** [comment, marked as answer]
No, such commands don't exist. Dialogs remember their location when moved elsewhere on the screen, though.
